forms/jqm/elements/choiceexpanded.js

```diff
@@ -18,27 +18,40 @@ define(function (require) {
       if (type !== 'select') {
         this.$el.find('input').off('click');
       }
+      this.$fieldset = null;
+
       return ChoiceElementView.prototype.remove.call(this);
     },
 
+    renderLabel: function () {
+      if (!this.$label) {
+        this.$label = $('<legend></legend>');
+      }
+
+      if (!this.$fieldset.find(this.$label).length) {
```

Comment:
What if you just used `jQuery.contains()` directly? http://api.jquery.com/jQuery.contains/